Readability.js

```diff
@@ -1292,12 +1310,81 @@ Readability.prototype = {
     });
   },
 
+  /**
+   * Try to extract metadata from JSON-LD object.
+   * For now, only Schema.org objects of type Article or its subtypes are supported.
+   * @return Object with any metadata that could be extracted (possibly none)
+   */
+  _getJSONLD: function (doc) {
+
```

Comment:
Nit: don't start/end blocks with empty lines.

I really need to take some time and adopt most of the Firefox eslint rules for this repo...

Reply:
Fixed!